back/db/schema.rb

```diff
@@ -1014,6 +1014,7 @@
 
   create_table "que_values", primary_key: "key", id: :text, force: :cascade do |t|
     t.jsonb "value", default: {}, null: false
+    t.check_constraint "jsonb_typeof(value) = 'object'::text", name: "valid_value"
```

Comment:
:grey_exclamation: Where is this coming from?

Reply:
Thanks, the migration bug I think It added a bunch of stuff. I missed this one. Nice catch.